node/runtime/pangolin/src/pallets/evm.rs

```diff
@@ -138,9 +139,11 @@ where
 			>>::execute(input, target_gas, context, is_static)),
 			a if a == addr(25) =>
 				Some(<Dispatch<R>>::execute(input, target_gas, context, is_static)),
-			a if a == addr(26) => Some(<StateStorage<R, StorageFilter>>::execute(
+			a if a == addr(27) => Some(<StateStorage<R, StorageFilter>>::execute(
```

Comment:
@wuminzhe I am going to update the `state-storage` precompile address of the pangolin network. See #1236 .